src/linear_algebra/clifford_algebra/conjugation.lean

```diff
@@ -71,7 +71,7 @@ by simp [reverse]
 by simp [reverse]
 
 @[simp] lemma reverse.map_one : reverse (1 : clifford_algebra Q) = 1 :=
-reverse.commutes 1
+by convert reverse.commutes (1 : R); simp
```

Comment:
This scares me a little - it sounds like `algebra_map R 1 (clifford_algebra Q)` is no longer defeq to `1`, which seems like a useful property to have.

Reply:
defeqness is nice when you can have it, but not really important IMHO: there are many situations where `algebra_map R 1 S` is not defeq to `1`, but it is not a problem since you can rewrite it away.

Reply:
For instance, I just checked and `algebra_map ℕ ℝ 1 = 1` is not definitional.

Reply:
Sure, but that's because the reals just underwent the same refactor! From reading the source alone, I think that both `matrix_algebra` and `triv_sq_zero_ext` satisfy this.